payments/vipps/after-payment.md

```diff
@@ -839,3 +742,4 @@ All Vipps error types will have the following URI in front of type:
 [technical-reference-reverse]: /payments/vipps/other-features#reversal
```

Comment:
Since we don't use the technical reference term anymore, maybe use other features instead?